torch/csrc/distributed/c10d/init.cpp

```diff
@@ -1375,6 +1375,11 @@ that adds a prefix to each key inserted to the store.
           .def("rank", &::c10d::ProcessGroup::getRank)
           .def("size", &::c10d::ProcessGroup::getSize)
           .def("name", &::c10d::ProcessGroup::getBackendName)
+          .def("id", &::c10d::ProcessGroup::getID)
+          .def(
+              "backend_id",
+              &::c10d::ProcessGroup::getBackendID,
+              py::arg("backend_type"))
```

Comment:
I wonder if these APIs must be exposed to end users?

Reply:
Added prefix.